test-data/unit/check-typeguard.test

```diff
@@ -639,7 +639,7 @@ if typeguard(n):
     reveal_type(n)
 [builtins fixtures/tuple.pyi]
 [out]
-main:4: error: invalid syntax
+main:4: error: invalid syntax; you likely need to run mypy using Python 3.8 or newer
```

Comment:
You can move these tests to https://github.com/python/mypy/blob/master/test-data/unit/check-python38.test